pkg/diagnostics/tracing_test.go

```diff
@@ -0,0 +1,65 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package diagnostics
+
+import (
+	"context"
+	"testing"
+
+	"github.com/dapr/dapr/pkg/channel"
+	"github.com/dapr/dapr/pkg/config"
+	"github.com/valyala/fasthttp"
+	"go.opencensus.io/trace"
+	"google.golang.org/grpc/metadata"
+)
+
+func TestTraceSpanFromFastHTTPRequest(t *testing.T) {
+	req := getTestHTTPRequest()
+	spec := config.TracingSpec{Enabled: true}
+
+	TraceSpanFromFastHTTPRequest(req, spec)
```

Comment:
This will do nothing because it will return anonymous function. We need to call this returned function for testing.

Since this code could be changed later. so I would just add `// TODO:` comment for now.

Reply:
> This will do nothing because it will return anonymous function. We need to call this returned function for testing.
> 
> Since this code could be changed later. so I would just add `// TODO:` comment for now.

No, this unit test is testing the code path for adding annotations, extracting the dapr-prefixed metadata.